Kickstarter-iOS/Library/UIImageView+URL.swift

```diff
@@ -17,6 +17,17 @@ extension UIImageView {
                      runImageTransitionIfCached: false,
                      completion: nil)
   }
+
+  public func ksr_roundedImageWith(_ url: URL) {
```

Comment:
One final thing I overlooked (apologies)...since this is a setter on `UIImageView` would it make sense to call this method `ksr_setRoundedImageWith(_ url: URL) { }` to imply that?

Reply:
fixed that 👍🏾